### DISCUSSION TITLE
GPL License

### DISCUSSION THREAD

**Discussion by diatredu2** [requarks/wiki#5864] [Help / Questions] [answered]
Hi, 

We would like to integrate your solution into our application. Since your solution has the GPL license, then it automatically means that our solution needs to be licensed as GPL, which is something we don't want to do. Is there any way to obtain your solution under a different license? Maybe there is some commercial license that will not oblige us to use GPL?

**Comment by NGPixel** [comment, marked as answer]
No plan on offering alternate licenses for now.
